### DISCUSSION TITLE
Login timeout error message is displayed twice

### DISCUSSION THREAD

**Discussion by emiliedwt** [keycloak/keycloak#26056] [Q&A]
I have a bug on the login page. When the login timeout is exceeded, I do get the error message displayed, but it appears twice.

<img width="549" alt="Screenshot 2024-01-09 at 17 32 33" src="https://github.com/keycloak/keycloak/assets/3437973/4a189f13-e2e4-419f-a35c-029d541d0d98">

This issue only occurs with this specific error message; other error messages display correctly. Have you ever encountered this problem?

keycloakVersion = "7.0.0"

**Comment by jonkoops** [comment]
You can likely fix this problem by upgrading to the latest version of Keycloak. The version you are running is extremely outdated and vulnerable to several critical security issues that can result in your installation getting compromised, so I would highly recommend it either way.

**Comment by emiliedwt** [comment]
Thank you for your response, and I'll follow your advice, I'll try to update keycloak to see if it fix my bug.
